modules/ROOT/pages/index-self-managed.adoc

```diff
@@ -45,15 +45,15 @@ Runtime Fabric on Self-Managed Kubernetes requires worker nodes which use the x8
 
 === Operating Systems
 
-Runtime Fabric on Self-Managed Kubernetes supports any Linux-based operating system supported by EKS or AKS.
+Runtime Fabric on Self-Managed Kubernetes supports any Linux-based operating system supported on EKS, AKS, or GKE.
 
 === Ingress Controller
 
-Runtime Fabric on Self-Managed Kubernetes supports any Ingress controller that is compatible with the Kubernetes environment on your vendor (EKS or AKS).
+Runtime Fabric on Self-Managed Kubernetes supports any Ingress controller that is compatible with the Kubernetes environment on your cloud infrastructure vendor.
```

Comment:
@sundippatel Do we want to explicitly call out GCP based ingress controller not supported for GKE? Or, modify the current statement to let customer tweaks configuration for specific ingress controller to fit their needs.

Reply:
Agree, we should modify to clarify ingress controllers exhibiting the behavior that GKE's default ingress controller is are not supported.

Reply:
Attempted to address; please review.
If able, we'd like to point to a KB article that we'll need to publish. Added a placeholder for now.